wallet/build.rs

```diff
@@ -28,8 +28,12 @@ fn main() {
 
     if let Ok(git_hash) = git_head_hash {
         println!("cargo:rustc-env=GIT_HEAD_HASH={}", git_hash);
+    } else {
+        println!("cargo:rustc-env=GIT_HEAD_HASH=");
```

Comment:
Not that important but "if else" can be replaced with
`println!("cargo:rustc-env=GIT_HEAD_HASH={}", git_hash.unwrap_or(""));`

Reply:
Done.